hapi-proto/src/main/proto/TransactionRecord.proto

```diff
@@ -45,4 +45,6 @@ message TransactionRecord {
     }
     TransferList transferList = 10; // All hbar transfers as a result of this transaction, such as fees, or transfers performed by the transaction, or by a smart contract it calls, or by the creation of threshold records that it triggers.
     repeated TokenTransferList tokenTransferLists = 11; // All Token transfers as a result of this transaction
+    uint32 tieOrder = 12; // Order of events with the same consensus timestamp
+    ScheduleID scheduleRef = 13; // Reference to the scheduled transaction ID that this transaction record represent
```

Comment:
The idea behind `ref` is to make it less confusing for clients. Let me explain.
We already have `scheduleID` in the receipt in the use-case when someone creates a new Scheduled Transaction. The newly created schedule is populated using `scheduleID` similarly to tokens/topics. 

This schedule id, populated in the `Transaction Record` is used to indicate **"the current transaction record was caused by that scheduled tx"**. If we add exactly the same `ScheduleID scheduleID` property in the `TransactionRecord.proto` for indicating that it might be misleading and clients might confuse it with the property used for creation.

Here we want to "reference" the scheduled tx id that caused the creation of this transaction record.